src/main/java/appeng/crafting/CraftingTreeNode.java

```diff
@@ -147,7 +147,8 @@ IAEItemStack request( final MECraftingInventory inv, long l, final IActionSource
 
 			for( IAEItemStack fuzz : itemList )
 			{
-				if( this.parent.details.isValidItemForSlot( this.getSlot(), fuzz.getItemStack(), this.world ) )
+				// TODO: check if OK
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1135)